test/data/stages/gzip/b.mpp.yaml

```diff
@@ -25,3 +25,4 @@ pipelines:
               sha256:f950375066d74787f31cbd8f9f91c71819357cad243fb9d4a0d9ef4fa76709e0: {}
         options:
           filename: compressed.gz
+          level: 10
```

Comment:
is `10` a valid gzip compression level? the man page seems to indicate `1-9`

Reply:
Oh I'm testing to see if my changes to the test manifest actually works by breaking it since CI just outputs "all 81 tests have passed", which doesn't really give much of an indicator.